package.json

```diff
@@ -34,8 +34,8 @@
     "cover 96% in us, not IE < 11"
   ],
   "scripts": {
-    "build": "rimraf dist && rollup -c rollup.config.js",
-    "build:test": "rimraf test-dist && rollup -c rollup.config.test.js",
+    "build": "rimraf dist && rollup -c rollup.config.ts --configPlugin '@rollup/plugin-typescript={tsconfig:`tsconfig.rollupConfig.json`}'",
+    "build:test": "rimraf test-dist && rollup -c rollup.config.test.ts --configPlugin '@rollup/plugin-typescript={tsconfig:`tsconfig.rollupConfig.json`}'",
```

Comment:
This command works only due to a luck. To make it work reliably, please add `rollup.config.test.ts` to the `include` section of `tsconfig.rollupConfig.json`.